documentation/docs/10-getting-started/25-project-types.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Project types
+---
+
+SvelteKit offers configurable rendering, which allows you to build and deploy your project in several different ways. You can build all of the below types of applications and more with SvelteKit. Rendering settings are not mutually exclusive and you may choose the optimal manner with which to render different parts of your application.
+
+If you don't have a particular way you'd like to build your application in mind, don't worry! The way your application is built, deployed, and rendered is controlled by which adapter you've chosen and a small amount of configuration and these can always be changed later.
+
+The project structure and routing will be the same regardless of the project type that you choose. If you are building an application with no backend or where the backend is running in a difference service, you can simply skip over and ignore the parts of the docs talking about `server` files.
+
+## Default hybrid rendered app
+
+By default, when a user visits a site, SvelteKit will render the first page with server-side rendering and subsequent pages with client-side rendering. Using SSR for the initial render improves SEO and perceived performance of the initial page load. Client-side rendering then takes over and updates the page without having to rerender common components which is typically faster and eliminates a flash when navigating between pages.
+
+## Static site generator (SSG)
```

Comment:
It bothers me that the format of these headers is different...

Should it be:
- Default: Hybrid rendering
- Static site generation (SSG)
- Single-page app (SPA)
?

Reply:
Even that doesn't feel quite right, struggling with how to align them

Reply:
I'm not sure what you mean that the format is different? Like you think they all need to end in "app"? The only one that feels a bit different from the rest to me is "Backend in another language", but I'm not sure how else to write that and think it's worth keeping given how many people want to do that and run into it

Reply:
Alright, after coming back to this with fresh eyes I think I see your point a bit more clearly. I believe I've cleaned this up now